File: Examples/VisualBasic/AsposePDF/DocumentConversion/PCLToPDF.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf
Namespace AsposePDF.DocumentConversion
    Public Class PCLToPDF
        Public Shared Sub Run()
            Try
                ' ExStart:PCLToPDF
                ' The path to the documents directory.
                Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_DocumentConversion()

                ' Instantiate LoadOption object using PCL load option
                Dim loadopt As LoadOptions = New PclLoadOptions()

                ' Create Document object
                Dim doc As New Document(dataDir & Convert.ToString("hidetext.pcl"), loadopt)

                ' Save the resultant PDF document
                ' ExEnd:PCLToPDF
                doc.Save(dataDir & Convert.ToString("PCLToPDF_out_.pdf"))
            Catch ex As Exception
                Console.WriteLine(ex.Message)
            End Try
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePDF/DocumentConversion/PCLToPDF.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.DocumentConversion
{
    public class PCLToPDF
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_DocumentConversion();

            // instantiate LoadOption object using PCL load option
            Aspose.Pdf.LoadOptions loadopt = new Aspose.Pdf.PclLoadOptions();
            
            // create Document object
            Aspose.Pdf.Document doc = new Aspose.Pdf.Document(dataDir + "test.pcl", loadopt);
            
            // save the resultant PDF document
            doc.Save(dataDir + "test-converted.pdf");
        }
    }
}